src/utils/helpers.rs

```diff
@@ -421,3 +429,20 @@ pub fn install_forge() -> Result<(), Box<dyn Error>> {
         Ok(())
     }
 }
+
+
+pub fn install_nargo() -> Result<(), Box<dyn Error>> {
```

Comment:
let the function name be `install_noir` nargo is what comes with noir installation just like cargo comes with rust installation.